static/src/javascripts/projects/common/modules/experiments/tests/commercial-lazy-loading.js

```diff
@@ -2,14 +2,14 @@
 import config from 'lib/config';
 
 export const commercialLazyLoading: ABTest = {
-    id: 'CommercialLazyLoading',
+    id: 'CommercialLazyLoadingExtended',
     start: '2018-02-16',
     expiry: '2018-03-30',
     author: 'Jon Norman',
     description:
         'This test alters the threshold for when lazy loaded adverts are displayed',
-    audience: 0.2,
-    audienceOffset: 0.7,
+    audience: 0.24,
+    audienceOffset: 0.4,
```

Comment:
Why did you choose these values?

Reply:
in the old test, there was an audience of 20%, and 5 variants. Each variant is 4% in size. So if I add another variant, I wanted the audience size to stay the same. Furthermore, it's a new test, so I chose an offset to ensure the new test audience doesn't overlap with the old test.